<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statisticsTable</class>
 <widget class="QDialog" name="statisticsTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidgetContainer">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>按照节点统计</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableByNode">
         <column>
          <property name="text">
           <string>节点名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>节点数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>错误数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>准确率</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>按天统计</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableByDate">
         <column>
          <property name="text">
           <string>日期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>节点数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>错误数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>准确率</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>按机位统计</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableByPosition">
         <column>
          <property name="text">
           <string>机位</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>节点数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>错误数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>准确率</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>按昼夜统计</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="tableByTime">
         <property name="rowCount">
          <number>2</number>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <item row="0" column="0">
          <property name="text">
           <string>白天</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>晚上</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>节点名</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>节点数</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>错误数</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>准确率</string>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>节点名</string>
          </property>
         </item>
         <item row="1" column="5">
          <property name="text">
           <string>节点数</string>
          </property>
         </item>
         <item row="1" column="6">
          <property name="text">
           <string>错误数</string>
          </property>
         </item>
         <item row="1" column="7">
          <property name="text">
           <string>准确率</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
